bundles/org.openhab.binding.sensorpush/README.md

```diff
@@ -0,0 +1,74 @@
+There is no easy way for the user to determine the sensor IDs in advance, so it is best to simply auto-discover them.
+After configuring the bridge, all active sensors should appear in the discovery inbox.
+
+### cloudbridge thing
+
+The `cloudbridge` thing is responsible for communicating with the SensorPush Gateway Cloud.
+You must supply your user name and password.
+The poll and timeout parameters are optional.
+
+Parameters:
+
+* `user` (required) Your SensorPush cloud service user name (email address)
+* `password` (required) Your SensorPush cloud service password
+* `poll` (2-60, default = 5) Polling interval in minutes
+* `timeout` (1-120, default = 30) Timeout period for API requests in seconds.
+
+**Note:** To activate your API access, you must log in to the [Gateway Cloud Dashboard](https://dashboard.sensorpush.com/) and agree to the terms of service.
+Once you've logged in that initial time, your account will have access.
+
+### sensor thing
+
+The `sensor` thing represents an individual SensorPush sensor.
+It has a variety of channels that will be populated with the latest sensor readings at each poll period.
+Sensors relay their readings at approximate 1 minute intervals, so in normal operation the oldest a reading should be is approximately 1 minute plus the configured poll interval.
+The `time` channel will contain the timestamp of the latest readings.
+If your particular sensor model does not support a given channel, the value for that channel will remain NULL.
+The id parameter must be supplied.
+
+Parameters:
+
+* `id` (required) The unique ID number of the sensor
+* `pressureMode` The reporting mode for barometric pressure. Must be set to either "station" or "meteorological". The station mode reports the pressure as recorded by the sensor, while the meteorological mode adjusts the reported pressure to mean sea level as is common for weather reporting. Defaults to "station".
+* `altitude` The altitude of the sensor in feet above MSL. It is only necessary to set this parameter if you selected the "meteorological" option for pressureMode and have not set the sensor altitude in the SensorPush app. The altitude set in the SensorPush app will override this value.
+
+## Channels
+
+The following channels are provided by the `sensor` thing:
+
+| channel     | type                     | description              |
+|-------------|--------------------------|--------------------------|
+| temperature | Number:Temperature       | Temperature              |
+| humidity    | Number:Dimensionless     | Relative humidity        |
+| pressure    | Number:Pressure          | Barometric pressure      |
+| dewpoint    | Number:Temperature       | Dew point                |
+| vpd         | Number:Pressure          | Vapor pressure deficit   |
+| time        | DateTime                 | Time of reading          |
+| rssi        | Number:Power             | Received signal strength |
```

Comment:
Yes, I looked at that when I defined the channel. But system.signal-strength is defined as a number 0-4, which didn't seem very useful when the hardware was providing RSSI in dBm. That's probably good for wifi signal levels which are often provided in some arbitrary hardware-specific units like 0-255, but I thought it better to leave it in dBm rather than trying to make up a mapping.

Reply:
You'd leave it up to the user to interpret the dBm value. Normal users don't know whether a specific dBm value is good or bad. That's the intention behind the system channel with its four values.

Reply:
I have a commit in another branch that adds a second rssi channel that maps the dBm value to a 0-4 value compatible with system.signal-strength. I don't want to merge it in to this branch before I tweak it a bit, though. The SensorPush app shows both the dBm value and 0-4 signal strength "bars", so I want to make sure the two match up first.

Reply:
Ok. I've merged the change into this branch. There are now 2 RSSI-related channels. One presents the RSSI in dBm and the other maps it to 1-4 signal strength bars. The mapping is consistent with the signal strength bars displayed in the SensorPush app, which shows both bars and the dBm number.